frontend/src/components/forms/input/inputTagsAutocomplete.tsx

```diff
@@ -71,22 +71,34 @@ export const InputTagsAutocomplete: React.FC<Props> = ({ suggestions = [], name
   };
 
   const addTag = (tag: any) => {
-    const tags = uniqBy([...selectedTags, tag], 'id');
-    setInputText('');
-    setSelectedTags(tags);
-    parentFormContext.setValue(name, tags);
+    if (tag) {
+      const tags = uniqBy([...selectedTags, tag], 'id');
+      setInputText('');
+      setSelectedTags(tags);
+      parentFormContext.setValue(name, tags);
+    }
   };
 
   const inputTag: ModelsSimpleTag = { id: inputText, type: 'SimpleTag' };
 
-  const displayCreateTag = inputText.length > 0 && !selectedTags.find((s) => s.id === inputText);
+  const displayCreateTag =
+    inputText.length > 0 && !selectedTags.find((s) => s && s.id === inputText);
```

Comment:
Could be `s?.id === inputText`? (nitpickTM)

Reply:
as I'm not adding undefined tags above s should always be defined, right?
```
 if (tag) {
      const tags = uniqBy([...selectedTags, tag], 'id');
      setInputText('');
      setSelectedTags(tags);
      parentFormContext.setValue(name, tags);
    }
```